manager/aica_django/connectors/NetworkScan.py

```diff
@@ -22,15 +22,15 @@
 from typing import Any, Dict
 
 from aica_django.connectors.DocumentDatabase import AicaMongo
-from aica_django.converters.Knowledge import nmap_scan_to_knowledge
+from aica_django.microagents.knowledge_base import record_nmap_scan
```

Comment:
Can you explain why this was renamed?

Reply:
To work in the same style as the other processors for example Netflow and offline loader, they do an apply_async.
Also in this part the knowledge also need to be recorded in Neo4J, herefor we have records_nmap_scan. The other record funtions also manage the xxx_to_knowledge part. So this is the best place. Before this the knowledge of nmap was not even stored in neo4j, this fixes the issue also.